rotkehlchen/tasks/calendar.py

```diff
@@ -410,6 +422,57 @@ def maybe_create_airdrop_claim_reminder(self) -> None:
             error_msg='Failed to add the airdrop claim reminders',
         )
 
+    def maybe_create_l2_bridging_reminder(self) -> None:
+        """Creates calendar reminders for L2 bridge claims (7 days after deposit).
+        Tracks deposits on Base, Optimism, and Arbitrum networks.
+        """
+        bridge_events: list[EvmEvent] = []
+        locations_to_counterparties = (
+            (Location.BASE, CPT_BASE),
+            (Location.OPTIMISM, CPT_OPTIMISM),
+            (Location.ARBITRUM_ONE, CPT_ARBITRUM_ONE),
+        )
+        db_history_events = DBHistoryEvents(database=self.database)
+        with self.database.conn.read_ctx() as cursor:
+            for location, counterparty in locations_to_counterparties:
+                bridge_events.extend(db_history_events.get_history_events(
+                    cursor=cursor,
+                    has_premium=True,
+                    filter_query=EvmEventFilterQuery.make(
+                        and_op=True,
+                        location=location,
+                        counterparties=[counterparty],
+                        event_types=[HistoryEventType.DEPOSIT],
+                        event_subtypes=[HistoryEventSubType.BRIDGE],
+                    ),
+                ))
+
+        now = ts_now_in_ms()
```

Comment:
you don't reuse this. Just don't use a variable

Reply:
<img width="602" alt="image" src="https://github.com/user-attachments/assets/6dd0f26d-d620-4819-a8b9-ca1c930347d1">